.github/workflows/ucode.yml

```diff
@@ -0,0 +1,62 @@
+name: Refresh Intel µcode submodule
+
+on:
+  schedule:
+    # At 23:35 on every day-of-week from Sunday through Saturday
+    # https://crontab.guru/#35_23_*_*_0-6
+    - cron: '35 23 * * 0-6'
+  workflow_dispatch:
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Code
+      uses: actions/checkout@v4
+
+    - name: Check if µcode submodule is up to date
+      run: |
+        git submodule update --init --checkout 3rdparty/intel-microcode
+        pushd 3rdparty/intel-microcode
+        current=$(git log -1 --pretty=format:"%H")
+        git checkout main
+        new=$(git log -1 --pretty=format:"%H")
+        if [[ $current == $new ]]; then
+          echo "Intel µcode submodule is up-to-date."
+        else
+          echo "Intel µcode submodule is out of date!"
+          exit 1
+        fi
+        popd
```

Comment:
I'd move this just after `new=$...`. It probably doesn't matter, but in theory it should also be done before `exit 1` above.

Reply:
Done in the latest rev: https://github.com/Dasharo/coreboot/compare/6088894b2ba051b1c5dc682c1c84ff7aa6a2b05b..638fbe41fc6e8e1d00653c6523d63b5acd11905b